src/lv_widgets/lv_keyboard.c

```diff
@@ -37,40 +37,80 @@ static lv_signal_cb_t ancestor_signal;
 static const char * const default_kb_map_lc[] = {"1#", "q", "w", "e", "r", "t", "y", "u", "i", "o", "p", LV_SYMBOL_BACKSPACE, "\n",
                                                  "ABC", "a", "s", "d", "f", "g", "h", "j", "k", "l", LV_SYMBOL_NEW_LINE, "\n",
                                                  "_", "-", "z", "x", "c", "v", "b", "n", "m", ".", ",", ":", "\n",
-                                                 LV_SYMBOL_CLOSE, LV_SYMBOL_LEFT, " ", LV_SYMBOL_RIGHT, LV_SYMBOL_OK, ""
+                                                 LV_SYMBOL_CLOSE, 
+                                                 #if LV_USE_ARABIC_PERSIAN_CHARS == 1
```

Comment:
Nitpick: please align preprocessor comments to the leftmost column.

Reply:
ok, on it

Reply:
I did it but now I think it was more readable as it was first
especially in the arrays